scripts/mkcloudhost/cloudfunc

```diff
@@ -45,3 +45,10 @@ function routedcloudpublicnet()
     base2=$(($base2 % 256))
     echo -n 10.$base1.$(($base2+$n-1))
 }
+
+function ipv6prefix()
+{
+    local n=$1
+    local mkch_id=$(mkch_id)
+    echo -n 'fd00:0::$mkch_id:$n'
```

Comment:
We wanted a /64 prefix. But 'fd00:0::$mkch_id:$n' would translate to fd00:0:0:0:0:0:$mkch_id:$n.. meaning the /64 would end up being "fd00:0:0:0" so don't you want to move the :: to the end?
  fd00:$mkch_id:$n:0::0 

So the /64 prefix is 'fd00:$mkch_id:$n:0'. That was when SLAAC is used, the last 64 bits can be used for the static mac-> ip it generates?

Reply:
Actually, https://study-ccna.com/ipv6-unique-local-addresses/ says, the upper 40 bits following "FD" are supposed to be randomly generated, so I did:
```bash
perl -e 'printf "%010X", rand(2**40)'
3BF4959651
```
And the next 16 bits are the "subnet ID"
That means, this would be appropriate
```bash
printf FD3B:F495:9651:%X%02X $mkch_id $n
```

Reply:
Hmmm, I was sort of hoping for a static configuration -- if we do random fd... addresses, then all virtual bridges will get renumbered every time we run the script, possibly impacting existing test environments.

Perhaps we should just ignore this PR and do this statically on mkch* by carving up 2620:113:80c0:8130::/64 (netblock from bond0 on mkchp)

@bmwiedemann What do you think?

Reply:
The suggested prefix `FD3B:F495:9651:` is a static configuration. I read that as you pick it once and hard code it in the script, not generate a new one every run.